backend/handlers/handlers.go

```diff
@@ -888,7 +888,7 @@ func (handlers *Handlers) getExchangeMoonpayBuy(r *http.Request) (interface{}, e
 	// TODO: Refactor to make use of a map.
 	var acct accounts.Interface
 	for _, a := range handlers.backend.Accounts() {
-		if a.Config().Code == acctCode {
+		if string(a.Config().Code) == acctCode {
```

Comment:
Or do the opposite?

```go
acctCode := accounts.Code(mux.Vars(r)["code"])
...
if a.Config().Code == acctCode {
   ...
}
```

Less conversions, easier to read. Here and in the other place.

Reply:
:+1: done